### PR TITLE
Add instructions for CI Visibility buddy integration

### PR DESCRIPTION
**Pull request by cg14823**
### What does this PR do?
Add instructions on how to setup the CI Visibility buddy integration.

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/continuous_integration/pipelines/buddy.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+<code>https://webhook-intake.{{< region-param key="dd_site" >}}/api/v2/webhook/?dd-api-key=<API_KEY>&env=ci&service=buddy-pipelines</code>
+
+### Including the repository
+
+Buddy does not include their repository URL in their webhooks. To get the most out of CI Visibility
+we recommend you add an action with name `[DD] GIT Reposository` that runs
+
+```
+echo "[DD] $BUDY_SCM_URL
```
Comment (andrea-mosk) [inline content/en/continuous_integration/pipelines/buddy.md]:
```suggestion
echo "[DD] $BUDY_SCM_URL"
```

File: content/en/continuous_integration/pipelines/buddy.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+2. Click **Save changes** to save the new webhook.
+
+### Setting custom tags
+To set custom tags to all the pipeline and job spans generated by the integration, add to the **Payload URL** a URL-encoded query parameter `tags`, with `key:value` pairs separated by commas. If a key:value pair contains any commas, surround it with quotes. For example, to add `key1:value1,"key2: value with , comma",key3:value3`, the following string would need to be appended to the **Webhook URL**:
+
+`?tags=key1%3Avalue1%2C%22key2%3A+value+with+%2C+comma%22%2Ckey3%3Avalue3`
+
+### Setting custom environment and service
+To set a custom environment and serivce for pipelines using the integration include a URL-encoded query parameter
```
Comment (juan-fernandez) [inline content/en/continuous_integration/pipelines/buddy.md:38]:
```suggestion
To set a custom environment and service for pipelines using the integration include a URL-encoded query parameter
```

File: content/en/continuous_integration/pipelines/buddy.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+in the **Payload URL**. For the environment this query parameter is `env` and for the service it is `service`. 
+For example to assign a custom environment `ci` and a service `buddy-pipelines` you would use the URL:
+
+<code>https://webhook-intake.{{< region-param key="dd_site" >}}/api/v2/webhook/?dd-api-key=<API_KEY>&env=ci&service=buddy-pipelines</code>
+
+### Including the repository
+
+Buddy does not include their repository URL in their webhooks. To get the most out of CI Visibility
+we recommend you add an action with name `[DD] GIT Reposository` that runs
```
Comment (juan-fernandez) [inline content/en/continuous_integration/pipelines/buddy.md:47]:
```suggestion
we recommend you add an action with name `[DD] GIT Repository` that runs
```

**Review by juan-fernandez — APPROVED** [review]
💪

**Comment by cg14823** [comment]
@andrea-mosk is it cool for me to close this? It keeps poping up in my GitHub and it is bugging me a bit, not sure if you guys still need it though 😄

**Pull request closed without merging**